lib/rubocop/cop/mixin/minitest_exploration_helpers.rb

```diff
@@ -43,6 +44,10 @@ def test_cases(class_node)
         test_cases + test_blocks
       end
 
+      def test_case_signature?(def_node)
```

Comment:
`test_method?` might be better, AFAIK signature doesn't include scope.

```suggestion
      def test_method?(def_node)
```